editor.planx.uk/src/components/Header.test.tsx

```diff
@@ -108,7 +109,7 @@ describe("Header Component - Editor Route", () => {
 
   it("displays breadcrumbs", () => {
     setup(<Header />);
-    expect(screen.getByText("Plan✕")).toBeInTheDocument();
+    // expect(screen.getByText("Plan✕")).to.be.
```

Comment:
Oops this looks like a mistake! Also should be able to comment-back-in a11y tests in this file - looks like I missed these ones!

Reply:
Thanks - not sure if this is a merge conflict issue or one that's been missed, but will resolve now!